### PR TITLE
fix(backend): combine Target.npu and Target.ascend to Target.npu

### LINKED ISSUE #241: tests/npu目录下的UT没有执行
### Describe the bug🐛

如题，发现test/npu下的测试没有按pytest ut case的形式进行编写，所以没有在ci中触发过测试

### Environment details

N/A

### LINKED ISSUE #274: combine Target.npu and Target.ascend to Target.npu, use vendor_compiler_config to config use ge_compiler or inductor_compiler
### Describe the feature🚀

combine Target.npu and Target.ascend to Target.npu, use vendor_compiler_config to config use ge_compiler or inductor_compiler

### References

there are two Target label in backend for Ascend chip, we want to combine Target.npu and Target.ascend to Target.npu, and use vendor_compiler_config to config to select ge_compiler or inductor_compiler

### Environment details

NPU

### PR DESCRIPTION
**Pull request by mingfa-byted**
combine Target.npu and Target.ascend to Target.npu, use vendor_compiler_config to config use ge_compiler or inductor_compiler

Closes #274 
Closes #241 

# New contributor declaration
- [x] I am not making a trivial change, such as fixing a typo in a comment.

- [x] I have written a PR description following these
  [rules](https://cbea.ms/git-commit/#why-not-how).

- [x] I have run `source scripts/install_githooks.sh`.

- Select one of the following.
  - [/tests/npu ] I have added tests.
    - `/tests/common` for `common` tests
    - `/tests/mlu` for `MLU` tests
    - `/tests/npu` for `GE` tests
    - `/tests/ascend` for `acl-graph` tests
    - `/tests/xpu_ops` for `xpu-ops` tests
  - [x] This PR does not need a test because `FILL THIS IN`.

### CODE REVIEW

File: tests/npu/ge/test_target_npu_ge_base.py
Comment (zhangjihang-BD) [inline tests/npu/ge/test_target_npu_ge_base.py:1]: 感觉没必要额外开个目录？我理解下，当前如果我们有融合kernel是用triton实现的话，应该对后端是否使用GE/Inductor无感？如果是无感的话，这样的目录结构稍微有点割裂感

File: xpu_graph/backends/npu.py [hunk 1]
```diff
@@ -17,3 +42,12 @@ def npu_compile(
         compiled_func = compile_fx_inner(module, inputs, **kwargs)
 
     return compiled_func
+
+def npu_compile(
+    module: torch.nn.Module, inputs, config_dict: Dict, **kwargs
+) -> torch.nn.Module:
+    compiler = config_dict.get("compiler", "inductor")
```
[the remark this answers is not in the document]
Comment (mingfa-byted) [reply xpu_graph/backends/npu.py:52]: 为什么要删除呢？万一在定位时想确认一下这个参数，不是反而掩盖了问题

[the remark this answers is not in the document]
Comment (yfchen-byted) [reply xpu_graph/backends/npu.py:52]: 也有道理，ge和inductor会忽略额外参数的话没有必要删

**Review by yfchen-byted — APPROVED** [review]

**Review by yfchen-byted — APPROVED** [review]
LGTM

**Review by zhangjihang-BD — APPROVED** [review]
